0.introduction/introduction.md

```diff
@@ -49,9 +49,9 @@ The utility that the Bitcoin blockchain delivers is _payments_, since that’s
 what users want to use the Bitcoin system for. To confuse the logical concept
 of a Bitcoin _transaction_ with the social concept of a payment is a
 conspicuous (and sadly very common) error. A _payment_ is a transfer of value
-from one party to another. In Bitcoin, a _transaction_is a transformation on
+from one party to another. In Bitcoin, a _transaction_ is a transformation on
 the global set of unspent transaction outputs, consuming some UTXOs and
-producing some new UTXOs.  _Transaction_ can also refer to a data structure
+producing some new UTXOs. A _Transaction_ can also refer to a data structure
```

Comment:
I think this change is a mistake.  I think it's intentional that the sentence start with the word "Transaction".  (The other changes LGTM, thanks!)

Reply:
I prefer the original as well.